python/GafferUI/ColorChooser.py

```diff
@@ -611,6 +614,31 @@ def setErrored( self, errored ) :
 	def getErrored( self ) :
 		return any( w.getErrored() for w in self.__numericWidgets.values() )
 
+	def setVisibleComponents( self, components ) :
+
+		self.__setVisibleComponentsInternal( components )
+
+	def getVisibleComponents( self ) :
+
+		return "".join( [ k for k, v in self.__sliders.items() if v.getVisible() ] )
+
+	def setStaticComponent( self, component ) :
```

Comment:
Is `StaticComponent` meaningful enough in the context of ColorChooser? Should it be `colorFieldStaticComponent()`?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/6003/commits/680a97c9d5bdd91088b7ab19e7d07d9c93d55040